developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler.ts

```diff
@@ -50,4 +53,45 @@ describe('keyboard-info-compiler', function () {
 
     assert.deepEqual(actual, expected);
   });
+
+  // ERROR_FileDoesNotExist (.kmp fileSize)
+
+  it('should generate FileDoesNotExist error if .kmp file does not exist', async function() {
+    const jsFilename = makePathToFixture('no-kmp', 'build', 'khmer_angkor.js');
+    const kpsFilename = makePathToFixture('no-kmp', 'source', 'khmer_angkor.kps');
+    const kmpFilename = makePathToFixture('no-kmp', 'build', 'khmer_angkor.kmp');
+
+    const sources = {
+      kmpFilename,
+      sourcePath: 'release/k/no-kmp',
+      kpsFilename,
+      jsFilename: jsFilename,
+      forPublishing: true,
+    };
+
+    const compiler = new KeyboardInfoCompiler();
+    assert.isTrue(await compiler.init(callbacks, {sources}));
+    compiler['fillLanguages'] = stubFillLanguages;
```

Comment:
Low priority, But could we just make this inline -- and add a comment?
```suggestion
    // stubbing fillLanguages internal function to avoid pulling in font resources in fixture
    compiler.fillLanguages. = async (_kpsFilename: string, _keyboard_info: KeyboardInfoFile, _kmpJsonData:  KmpJsonFile.KmpJsonFile): Promise<boolean> => true;
```